extensions/ql-vscode/src/vscode-tests/no-workspace/run-queries.test.ts

```diff
@@ -47,9 +46,13 @@ describe('run-queries', () => {
       const qs = createMockQueryServerClient();
       const mockProgress = 'progress-monitor';
       const mockCancel = 'cancel-token';
+      const mockQlProgram = {
+        mock: 'program'
+      } as unknown as QlProgram;
```

Comment:
I'd personally make a more proper mock to help with debugging.

```ts
const mockQlProgram = {
  dbschemePath: '',
   libraryPath: [],
   queryPath: ''
} as QlProgram;
```

Feel free to ignore though!

Reply:
That's fine. I was just being lazy. :)